src/text_shaper/open_shaper.cpp

```diff
@@ -477,7 +475,7 @@ namespace // {{{ helper
     }
 } // }}}
 
-struct FontInfo
+struct HbFontInfo
```

Comment:
The rename is here is a must, because under `dwrite_shaper` there's struct with exact same name & namespace, which ruins the stack with mixing usage of shaper & engine.

Reply:
absolutely fine with me.